services/cinema_hall.py

```diff
@@ -0,0 +1,17 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> CinemaHall.objects:
```

Comment:
I think it is better to `import` and use `QuerySet` instead of `CinemaHall.objects`

Reply:
Thanks for your advice.